documentation/framework/docs/ripple.py

```diff
@@ -0,0 +1,717 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 John Gabriele <jgabriele@fastmail.fm>
+#
+# This file constitutes the Rippledoc program.
+#
+# Rippledoc is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published
+# by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# Rippledoc is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Rippledoc.  If not, see <http://www.gnu.org/licenses/>."
```

Comment:
Can this rename `rippledoc.py` in line with the officially maintained version [here](https://gitlab.com/uvtc/rippledoc)?

It is also GPL-3 so it license file may need to be referenced or pulled in to comply.

I wonder if we might be able to wrap this in a python package or possibly offer an upstream PR and request the maintainers publish an official package that would allow users to simple run a `requirements.txt` install to get `rippledoc` on their path? I don't see a lot of activity upstream, however would be nice to keep in sync from a package vs vendor in our repo.

Reply:
I like the idea of bringing the tool to users quickly, it may also be reasonable to document this tools installation usage or add a script in `tools` that will pull the latest ensure the python env and run it against the local repo.

Reply:
> It is also GPL-3 so it license file may need to be referenced or pulled in to comply.
I wasn't sure how to do this. I was going to bring it up later. Any guidance is appreciated. 

> Can this rename rippledoc.py in line with the officially maintained version here?
Done!

I noticed there wasn't much activity, which doesn't bother me too much. Pandoc doesn't move quickly so the script should work for some time. I'll try and reach out to see if I can get the maintainer to make it a package.

Reply:
The PR is getting big, so I really want to seperate the md and html. Maybe have just the HTML files in the repo with instructions on editing / contributing.